Source/RapyutaSimulationPlugins/Private/Tools/RRROS2TFPublisher.cpp

```diff
@@ -5,6 +5,11 @@
 // rclUE
 #include "Msgs/ROS2TFMsg.h"
 
+URRROS2TFPublisher::URRROS2TFPublisher()
+{
+    PublicationFrequencyHz = 50;

```

Comment:
Need `MsgClass = UROS2TFMsg::StaticClass();`? in ctor also?
(Just a note, about `IsStatic`, it might be reset after ctor & before `InitializeWithROS2()` so it should not be checked in ctor)

Reply:
Move `MsgClass = UROS2TFMsg::StaticClass();` here to ctor also?
(About `IsStatic`, it might be reset after ctor & before `InitializeWithROS2()` so it should not be checked in ctor)